src/components/gltf-model-plus.js

```diff
@@ -136,10 +136,11 @@ const inflateEntities = function(node, templates, isRoot) {
     z: node.rotation.z * THREE.Math.RAD2DEG
   });
   el.setAttribute("scale", {
-    x: node.scale.x,
-    y: node.scale.y,
-    z: node.scale.z
+    x: node.scale.x * (scale !== undefined ? scale : 1),
```

Comment:
At first glance I thought this was going to scale all the nodes, but then i realized its just not passed through on recursion, probably ok got unexpected at first